scripts/bundleTrackers.mjs

```diff
@@ -0,0 +1,25 @@
+import fetch from 'node-fetch'
+import { writeFileSync } from 'fs'
+
+const tdsUrl = 'https://staticcdn.duckduckgo.com/trackerblocking/v4/tds.json'
+const resp = await fetch(tdsUrl)
+const tds = await resp.json()
+
+// Build a trie of tracker domains, starting with the broadest subdomain. Leaves are set to 1 to indicate success
+// i.e. lookup['com']['example'] === 1 if example.com is a tracker domain
+const trackerLookupTrie = {}
+function insert (domainParts, node) {
+    if (domainParts.length === 1) {
+        node[domainParts[0]] = 1
+    } else if (node[domainParts[0]]) {
+        insert(domainParts.slice(1), node[domainParts[0]])
+    } else {
+        node[domainParts[0]] = {}
+        insert(domainParts.slice(1), node[domainParts[0]])
+    }
+}
+Object.keys(tds.trackers).forEach((tracker) => {
+    insert(tracker.split('.').reverse(), trackerLookupTrie)
+})
+
+console.log(JSON.stringify(trackerLookupTrie))
```

Comment:
as mentioned in another comment, just write directly to disk within node here and it'll work on windows then

Reply:
Kept this for the extension but allowed for --output in the default case here for C-S-S